video/out/vo_gpu_next.c

```diff
@@ -84,6 +84,17 @@ struct user_lut {
     struct pl_custom_lut *lut;
 };
 
+struct frame_info {
+    int count;
+    struct pl_dispatch_info info[VO_PASS_PERF_MAX];
+};
+
+struct perf_data {
```

Comment:
Nit: this could be an anonymous struct inside `struct priv`, it is never re-used anywhere else

(Or even not exist)

Reply:
Done